src/frequenz/sdk/timeseries/formula_engine/_formula_generators/_formula_generator.py

```diff
@@ -42,13 +44,75 @@ class ComponentNotFound(FormulaGenerationError):
 """
 
 
+class FallbackFormulaMetricFetcher(FallbackMetricFetcher[QuantityT]):
```

Comment:
I'm guessing this is done because of circular imports?  If so adding a comment would be helpful.

Still maybe it would be nicer to put this new class in a separate file like `_fallback_components.py` rather than in `_formula_generator.py`

Reply:
Moved to separate file : src/frequenz/sdk/timeseries/formula_engine/_formula_generators/_fallback_formula_metric_fetcher.py